kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/spec/style/scopes/ShouldSpecContextScope.kt

```diff
@@ -43,6 +45,26 @@ class ShouldSpecContextScope(
       }
    }
 
+   fun context(name: String) =
+      ShouldSpecContextConfigBuilder(
+         createTestName(name),
+         description,
+         testContext,
+         defaultConfig.toTestContainerConfig(),
+         lifecycle,
+         false
+      )
+
+   fun xcontext(name: String) =
```

Comment:
We should add a `suspend` version of `xcontext` as well, like have added in `FunSpecContextScope`.

Reply:
Thanks, done.